### PR TITLE
Add notification popup on game installed

### PR DESCRIPTION
**Pull request by orende**
## Description

Shows a native notification popup when games have been successfully downloaded and installed. Partially addresses #566 .

## Checklist
 
 - [x] _CHANGELOG.md_ was updated (**format**: - Change made (thanks to github_username))

### CODE REVIEW

File: minigalaxy/ui/gametile.py [hunk 1]
```diff
@@ -343,6 +343,10 @@ def __install(self, save_location, update=False, dlc_title=""):
                 self.game.set_dlc_info("version", self.api.get_version(self.game, dlc_name=dlc_title), dlc_title)
             else:
                 self.game.set_info("version", self.api.get_version(self.game))
+            popup = Notify.Notification.new("Minigalaxy", _("Finished downloading and installing {}")
+                                            .format(self.game.name), "dialog-information")
+            popup.show()
+            popup.close()
```
Comment (sharkwouter) [inline minigalaxy/ui/gametile.py:349]:
```suggestion
```
This seems to hide the notification.

**Review by sharkwouter — CHANGES_REQUESTED** [review]
Thanks a lot, that is such a cool change!

I made a small change, after that it worked well on Gnome. Without it the notifications would not show. Otherwise it looks good to me.

**Comment by orende** [comment]
@sharkwouter  There's still a `popup.close()` call in the gametilelist.py file, which I fear could cause the same problem you had with the gametile.py file. Do you want me to open up another PR to fix that?

**Comment by sharkwouter** [comment]
Oh, I forgot about that. Good call. Please do

**Pull request merged**